server/util/ServiceError.js

```diff
@@ -0,0 +1,9 @@
+class ServiceError extends Error {
```

Comment:
Not saying we shouldn't, but out of curiosity, why are we adding another error handles in addition to `apiError`?

Reply:
My understanding is that all thrown exceptions pass through `app.use(errorHandler);` in `server/index.js`, which is a wrapper for `exports.apiError`. `ServiceError` is just a class that produces uniform error objects, which I'd like to use anywhere that the server is currently using generic `Error` instances